yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -320,7 +328,12 @@ export class Sequencer {
       const pendingTxs = this.p2pClient.iteratePendingTxs();
 
       await this.buildBlockAndEnqueuePublish(pendingTxs, proposalHeader, newGlobalVariables).catch(err => {
-        this.log.error(`Error building/enqueuing block`, err, { blockNumber: newBlockNumber, slot });
+        if (err instanceof FormattedViemError) {
+          this.log.verbose(`Unable to build/enqueue block ${err.message}`);
```

Comment:
when we have the pretty error, just making it use it here instead of being all red and sad